app/delivery/send_to_providers.py

```diff
@@ -107,34 +105,32 @@ def send_email_to_provider(notification):
             values=notification.personalisation
         )
 
+        created_at = notification.created_at
         if service.research_mode or notification.key_type == KEY_TYPE_TEST:
             notification.reference = str(create_uuid())
             update_notification_to_sending(notification, provider)
             send_email_response(notification.reference, notification.to)
         else:
             from_address = '"{}" <{}@{}>'.format(service.name, service.email_from,
                                                  current_app.config['NOTIFY_EMAIL_DOMAIN'])
-
-            email_reply_to = notification.reply_to_text
-
             reference = provider.send_email(
                 from_address,
-                validate_and_format_email_address(notification.to),
+                notification.normalised_to,
                 plain_text_email.subject,
                 body=str(plain_text_email),
                 html_body=str(html_email),
-                reply_to_address=validate_and_format_email_address(email_reply_to) if email_reply_to else None,
+                reply_to_address=notification.reply_to_text,
```

Comment:
While this is probably OK, I'm having trouble finding proof that the text in here has been validated equivalently at a previous stage, or if that's even important. 

There are multiple potential origins (see [here](https://github.com/alphagov/notifications-api/blob/2270832873c9a44d9cc86fa40ee5d8e6ef618e4c/app/service/send_notification.py#L126)) and at least one of them does it's own validation ([here](https://github.com/alphagov/notifications-api/blob/2270832873c9a44d9cc86fa40ee5d8e6ef618e4c/app/service/service_senders_schema.py#L9)). It would be helpful to have a bit more explanation in the commit message about where this is set.

Reply:
The reply_to_text is validated when added to the service, I don't see the need to validate it each time the service sends an email.
I suspect the validation was there before we confirmed the validity of the email address from the admin app. I don't believe it is necessary now.

Reply:
OK. We do validate them when they are added. If that's the only way for this field to be populated, then I agree there's no need to validate at this point.